test/metabase/query_processor_test/filter_test.clj

```diff
@@ -78,6 +78,24 @@
         {:filter   [:!= $liked true]
          :order-by [[:asc $id]]}))))
 
+;;; filter != nil
+(qp.test/expect-with-non-timeseries-dbs
```

Comment:
According to this macro's docstring, it's deprecated. Is there maybe a non-obvious reason to still use it?

Reply:
Just symmetry with the others, but that's a poor reason. I'll change it